src/components/Vuetify/mixins/goTo.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Modified from https://github.com/alamcordeiro/vue-smooth-scroll
+ */
+
+function easeInOutCubic (t) { return t < .5 ? 4 * t * t * t : (t - 1) * (2 * t - 2) * (2 * t - 2) + 1 }
+
+export default function (target) {
+  if (typeof window === 'undefined') return
+
+  let duration = 500
+  let start = Date.now()
+
+  function step () {
+    let elapsed = Date.now() - start
```

Comment:
And here you should use the timestamp that is passed by `requestAnimationFrame`

https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame